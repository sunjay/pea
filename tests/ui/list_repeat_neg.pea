fn main() {
    let a = [b"abc"; -400];
}
