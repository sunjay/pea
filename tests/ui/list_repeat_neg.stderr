stack backtrace:
   0: main
error: list repeat length argument must be a non-negative number
